proto/dymension/eibc/tx.proto

```diff
@@ -1,19 +1,32 @@
 syntax = "proto3";
 package dymensionxyz.dymension.eibc;
 
+import "cosmos/msg/v1/msg.proto";
+
 option go_package = "github.com/dymensionxyz/dymension/v3/x/eibc/types";
 
 // Msg defines the Msg service.
 service Msg {
     rpc FulfillOrder(MsgFulfillOrder) returns (MsgFulfillOrderResponse) {}
+    rpc UpdateDemandOrder(MsgUpdateDemandOrder) returns (MsgUpdateDemandOrderResponse) {}
 }
 
 // MsgFulfillOrder defines the FulfillOrder request type.
 message MsgFulfillOrder {
+    option (cosmos.msg.v1.signer) = "fulfiller_address";
     // fulfiller_address is the bech32-encoded address of the account which the message was sent from.
     string fulfiller_address = 1;
     string order_id = 2;
```

Comment:
should we not also have the original order fee included here, for comparing with the current one, in case it was updated in the meantime?